src/asmcnc/apps/start_up_sequence/welcome_to_smartbench_app/screens/screen_language_select.py

```diff
@@ -411,6 +415,10 @@ def next_screen(self):
 		self.start_seq.next_in_sequence()
 
 	def update_strings(self):
+		if self.l.lang == self.l.ko: 
```

Comment:
The font is not set back to default (not a blocking issue though)

Reply:
fixed